src/openrct2/network/Socket.cpp

```diff
@@ -125,8 +125,8 @@ static bool InitialiseWSA()
 class SocketException : public std::runtime_error
 {
 public:
-    explicit SocketException(const std::string& message)
-        : std::runtime_error(message)
+    explicit SocketException(std::string message) noexcept
+        : std::runtime_error(std::move(message))
```

Comment:
This makes little sense since runtime_error has no constructor using rvalue, it will always end up as a copy so I'd prefer const reference since thats also what runtime_error expects.

Reply:
Reverted